src/modules/Lan.rb

```diff
@@ -866,14 +866,14 @@ def ProposeVirtualized
 
         bridge_name = format("br%s", NetworkInterfaces.GetFreeDevice("br"))
 
-        if !LanItems.IsBridgeable(bridge_name, current)
-          log.info "The interface #{ifcfg} is not bridgeable."
+        if !LanItems.IsBridgeable(bridge_name, current) || !proposable_as_bridge?(config)
+          log.info "The interface #{ifcfg} can not be proposed as bridge."
```

Comment:
np: but you don't propose ```ifcfg``` to be a bridge. You're checking if given device can be inserted into a bridge (as a slave)

Reply:
Yep, that is true, the naming was not correct as it was just proposing the item config when not configured and then it uses some properties for the bridge itself.